src/content/docs/style-guide/formatting/format-overview.mdx

```diff
@@ -11,5 +11,3 @@ redirects:
 Generally, follow the Microsoft Style Guide for [formating common text elements](https://docs.microsoft.com/en-us/style-guide/text-formatting/formatting-common-text-elements). 
 
 You can find code formatting info in the [code examples page of this guide](/docs/style-guide/formatting/code-examples).
```

Comment:
@austin-schaefer, this intro topic is a bit sparse, and I'm not sure why the code formatting topic is the only other internal link when we discuss a variety of other formatting.

Reply:
I'm puzzled by the existence of this doc as well. Thoughts on just deleting it?